solr/core/src/java/org/apache/solr/cli/ApiTool.java

```diff
@@ -56,24 +56,38 @@ public List<Option> getOptions() {
             .argName("URL")
             .hasArg()
             .required(true)
-            .desc("Send a GET request to a Solr API endpoint.")
-            .build());
+            .desc("Send a GET request to a Solr API endpoint, or path to use with the default Solr URL.")
+            .build(),
+        SolrCLI.OPTION_ZKHOST,
+        SolrCLI.OPTION_SOLRURL,
+        SolrCLI.OPTION_SOLRPORT,
+        SolrCLI.OPTION_SOLRHOST,
+        SolrCLI.OPTION_URLSCHEME,
+        SolrCLI.OPTION_VERBOSE);
   }
 
   @Override
   public void runImpl(CommandLine cli) throws Exception {
     String response = null;
     String getUrl = cli.getOptionValue("get");
     if (getUrl != null) {
```

Comment:
Do we need this check?   If the get parameter is marked required in the ocnfiguration, I would assume you wouldn't get to here if it is missing!